Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift

```diff
@@ -84,6 +84,11 @@ public class BTPayPalMessagingView: UIView {
             ])
         }
     }
+
+    private func notifyFailure(with error: Error) {
+        apiClient.sendAnalyticsEvent(BTPayPalMessagingAnalytics.failed, errorDescription: error.localizedDescription)
+        delegate?.onError(self, error: error)
+    }
```

Comment:
Adding analytics tests and noticed there was a bug with sending analytics _only_ for failures from the messaging SDK but not when we trigger an error. This is now fixed (and tested) in this PR.